spec/app/models/account.rb

```diff
@@ -25,4 +25,12 @@ class Account
   def overridden
     self[:overridden] = "not recommended"
   end
+
+  # MONGOID-3365
```

Comment:
I think someone can be confused by ```MONGOID-3365``` comment easily, since there's no chance to understand it without knowledge of some context (for example I know this is JIRA id).

If you want to keep reference for this change, it will be better to add direct link to JIRA or a simple text comment like ```searching by a Time value```.

But this is just my opinion.